module/basicfantasyrpg.mjs

```diff
@@ -94,7 +102,7 @@ Handlebars.registerHelper('toLowerCase', function(str) {
 });
 
 Handlebars.registerHelper('selected', function(value) {
-  return Boolean(value) ? "selected" : "";
+  return value ? "selected" : "";
```

Comment:
What was the rationale for this change? If I recall (and it was some time ago), I coerced to boolean here to ensure the return was a true or false value if the helper existed, otherwise it would return the empty string. Did you run into an issue with that logic?

Reply:
To be honest, I don't remember why. It's been a while I wrote this, but it looks like it would be safer to add the coercion back.

Reply:
I realize now that it was actually my IDE (WebStorm) automatically deleting it. I'm not very experienced in javascript, but it looks like the coercion in practice doesn't change anything.